tests/galsim_test_helpers.py

```diff
@@ -434,10 +420,6 @@ def do_pickle(obj1, func = lambda x : x, irreprable=False):
     assert object() != f1
 
     # Test the hash values are equal for two equivalent objects.
-    try:
-        from collections.abc import Hashable
-    except ImportError:
-        from collections import Hashable
     if isinstance(obj1, Hashable):
         #print('hash = ',hash(obj1),hash(obj2))
         assert hash(obj1) == hash(obj2)
```

Comment:
Below on lines 470-528:  This section appears to only barely get run anymore; just random.py features an explicitly coded `__getinitargs__`.  I'm guessing this was more useful in the past when the c++-wrapped classes were old-style in py27 maybe?  (Maybe only in boost_python?)

I believe py3 only features new-style classes, even when inheriting from `object`, so recommend global-search-replace to just make everything look "new-style".

Reply:
Interesting. Good point, although the point of that section was to check that our `__eq__` function were checking everything important.  So that means we don't really have a good check of that anymore.  I don't know if it's worth trying to do something clever with `inspect` maybe to try to get that same test functionality?  Probably not I guess.
I can report that the one place left that used `__getinitargs__`, which you pointed out, doesn't need it.  Removing that function keeps `DistDeviate` serializing just fine.  So that was some old workaround, probably from pre-2.0.  I deleted that, which means this test section now never gets run at all.
So what do you think, should I remove it as useless, or leave it with a note that the argument list it loops over would need to be changed to some other technique to make it useful again?

Reply:
Side point.  I also got rid of this little amusing DistDeviate method that I assume used to actually do something:
```
@property
    def _function(self):
        return self.__function if isinstance(self.__function, str) else self.__function
```
😆

Reply:
> I don't know if it's worth trying to do something clever with inspect maybe to try to get that same test functionality?

I'm not sure if there's anything in `inspect` that would take an object and reproduce its ctor args.  We might be able to perturb the items returned by `__getstate__`, but I'm not sure it's worth it.

We still have `all_obj_diff`, of course.  The downside is that we have to remember to use it instead of relying on objects being automatically tested.  So I vote to just remove this code.

Reply:
Also, I'm not sure I was very clear above, but by "recommend global-search-replace to just make everything look "new-style"" I meant to comb through the code and remove any inheritance from `object`, just to make the code look more modern.

Reply:
> remove any inheritance from object
Ah.  Good.  Yes, I'll do that.  I didn't pick up that intent the first time. :)